lib/view/formulier/InvoerVelden.class.php

```diff
@@ -796,55 +796,104 @@ class RequiredLidField extends LidField {
 
 /**
  * Select an entity based on primary key values in hidden input fields, supplied by remote data source.
+ *
+ * NOTE: support alleen entities met een enkele primary key.
  */
 class EntityField extends InputField {
 
-	private $show_value;
-
-	public function __construct($name, array $primary_key_values, $description, PersistenceModel $model, $show_value, $find_url = null) {
-		parent::__construct($name, $primary_key_values, $description, $model);
-		$this->show_value = $show_value;
-		$this->suggestions[] = $find_url;
-	}
-
-	public function validate() {
+	private $show_value, $entity, $orm;
+
+    /**
+     * @var PersistenceModel
+     */
+    protected $model;
+
+    /**
+     * EntityField constructor.
+     * @param $name string Prefix van de input
+     * @param $show string Attribuut van $model om in de input weer te geven
+     * @param $description string Beschrijvijng van de input
+     * @param PersistentEntity $entity
+     * @param PersistenceModel $model Model
+     * @param $url string Url waar aanvullingen te vinden zijn
+     */
+    public function __construct($name, $show, $description, PersistenceModel $model, $url) {
+        $this->orm = $model::ORM;
+        $this->entity = new $this->orm();
+        foreach ($this->entity->getPrimaryKey() as $key) {
+            $this->entity->$key = filter_input(INPUT_POST, $name . '_'. $key, FILTER_DEFAULT);
+        }
+
+        parent::__construct($name, $this->entity->$show, $description, $model);
+        $this->suggestions[] = $url;
+        $this->show_value = $show;
+    }
+
+    public function getName() {
+        // TODO: Dit moet de volledige pk kunnen zijn.
+        return $this->name . '_' . $this->entity->getPrimaryKey()[0];
+    }
+
+    public function getValue() {
+        // TODO: Dit moet de volledige pk kunnen zijn.
```

Comment:
Verwijder het hele multiple fields voor alle PK delen en gebruik simpelweg UUID. Dat is 1 string en elk Model heeft een retrieveByUUID. Simpeler en beter. Zie voorbeeld #92 https://github.com/csrdelft/csrdelft.nl/pull/92/files#diff-a084f49f447b776638226d747586b742R255